tests/layer_tests/tensorflow_tests/test_tf_NonMaxSupression.py

```diff
@@ -11,16 +11,6 @@
 
 
 class TestNonMaxSuppression(CommonTFLayerTest):
-
-    # overload inputs generation to suit NMS use case
-    def _prepare_input(self, inputs_dict):
-        channel = ':0' if not self.use_legacy_frontend else ''
-        input_data = {}
-        for input in inputs_dict.keys():
-            input_data[input + channel] = np.random.uniform(low=0, high=1,
-                                                            size=inputs_dict[input]).astype(np.float32)
-        return input_data
```

Comment:
why was it removed?

Reply:
I would ask you to generate data like for image from [0, 1] range.

Reply:
This code adds ':0' to input names for new frontend, but currently names already name ":0", so this method not needed and default _prepare_input() works.

Reply:
> I would ask you to generate data like for image from [0, 1] range.

Done.